### ISSUE TITLE
Amazon Linux Package (Agent)

### ISSUE THREAD

**Issue by ltickett** [netxms/packages-rpm#2]
Is there a package for Amazon Linux 2 or 2023?

I was able to build something https://gitlab.com/tickett/netxms-amazon-linux (requires manual “installation” as I’ve never built a Linux install package before)

But it surprises me there isn’t something already available.

Can I submit something upstream?

Where is the code that builds the rhel package for example?

Thanks a million

**Comment by alkk** [comment]
Hi.

I've added Amazon Linux target (both 2 and 2023) to our RPM builder, if everything will go according to plan, RPMs will be available shortly.

RPMs are built by drone using this pipeline: https://github.com/netxms/packages-rpm/blob/master/.drone.yml
Sources for the build container are here: https://github.com/netxms/builder-rpm

**Comment by ltickett** [comment]
Oh wow- thanks @alkk 

Is https://github.com/netxms/builder-rpm private? I'm getting a 404 trying to access it

**Comment by alkk** [comment]
Yes, it was private for some reason - I've changed visibility. Anyway, it's just a simple container with mock and some dependencies inside.

There is a slight issue with the RPMs - some packages (like libvirt or mosquitto) are missing from the Amazon Linux repositories. We build all packages from the single SPEC file, but I need to add conditions there (e.g. to exclude netxms-agent-mqtt).
